packages/record-data/addon/-private/relationships/state/relationship.ts

```diff
@@ -489,22 +515,18 @@ export default class Relationship {
     @private
    */
   removeCompletelyFromInverse() {
```

Comment:
Were we ever hitting this before?

Reply:
no, it was here just for typescript it seems.